onnx/cpp2py_export.cc

```diff
@@ -85,6 +136,10 @@ PYBIND11_MODULE(onnx_cpp2py_export, onnx_cpp2py_export) {
             return py::bytes(bytes);
           })
       .def_property_readonly("has_context_dependent_function", &OpSchema::HasContextDependentFunction)
+      .def("infer_node_outputs", CallNodeInferenceFunction,
+          py::arg("nodeBytes"), py::arg("valueTypesByNameBytes"),
+          py::arg("inputDataByNameBytes") = std::unordered_map<std::string, py::bytes>{},
```

Comment:
None is better than an empty container as a default value.

Reply:
What do you mean by None here? A null pointer? 
I think that not having any information in the map makes sense as a default for inference.

Reply:
I don't think I know what you mean here. This is C++ side, so there's no None. Do you mean to make it a pointer and use nullptr/use std::optional? 
I think that this wouldn't be a lot better as the map isn't mutated and also in this it makes sense that the default is "no additional information" (for the inference context we have to pass in _some_ map, and it makes sense to make it empty as there's no input data we know).
Could you provide a code example for the change?